workflows/treeval.nf

```diff
@@ -55,31 +45,61 @@ include { CUSTOM_DUMPSOFTWAREVERSIONS } from '../modules/nf-core/custom/dumpsoft
 */
 
 workflow TREEVAL {
+    take:
+    assembly_id     // channel:
+    reference       // channel:
+    map_order       // channel: hic mapping order (from yaml)
+    assem_reads     // channel: path to longreads directory (from yaml)
+    kmer_prof_file  // channel:
+    hic_reads       // channel: path to hic reads directory (from yaml)
+    supp_reads      // channel:
+    align_genesets  // channel: paths to genesets in from yaml
+    synteny_paths   // channel: path to syntenic genomes (from yaml)
+    intron_size     // channel:
+    teloseq         // channel: telomere motif sequence (from yaml)
+    lineageinfo     // channel:
+    lineagespath    // channel:
+
     main:
     //
     // PRE-PIPELINE CHANNEL SETTING - channel setting for required files
     //
-    ch_versions     = Channel.empty()
+    ch_versions         = Channel.empty()
+
+    exclude_steps_list = params.steps.length() > 1 ? params.steps.tokenize(',').collect { it.trim() } : params.steps
 
-    params.steps    = params.steps ?: 'NONE'
-    exclude_workflow_steps = params.steps.length() > 1 ? params.steps.split(',').collect { it.trim() } : params.steps
+    all_steps_list       = ["insilico_digest", "gene_alignment", "repeat_density", "gap_finder", "selfcomp", "synteny", "read_coverage", "telo_finder", "busco", "kmer", "hic_mapping", "NONE"]
 
-    full_list       = ["insilico_digest", "gene_alignment", "repeat_density", "gap_finder", "selfcomp", "synteny", "read_coverage", "telo_finder", "busco", "kmer", "hic_mapping", "NONE"]
+    jbrowse_exclude_list = ["insilico_digest", "gene_alignment", "selfcomp", "synteny", "busco", "kmer"]
+    rapid_exclude_list = ["gene_alignment", "repeat_density", "gap_finder", "read_coverage", "telo_finder", "hic_mapping"]
+    rapid_tol_exclude_list = ["gene_alignment", "repeat_density", "gap_finder", "read_coverage", "telo_finder", "kmer",  "hic_mapping"]
```

Comment:
please test all these exclude_list, for instance, gene_alignment should not be in the rapid mode.